assets/src/components/mapPage/mapDisplay.tsx

```diff
@@ -87,18 +97,26 @@ const onFollowerUpdate: UpdateMapFromPointsFn = (map, points) => {
     .getCenter()
     .subtract(mapContainerBounds.getCenter())
 
-  const targetZoom = 16
-  const targetPoint = map
-      // Project the target point into screenspace for the target zoom
-      .project(points[0], targetZoom)
-      // Offset the target point in screenspace to move the center of the map
-      // to apply the padding to the center
-      .subtract(offset),
-    // convert the target point to worldspace from screenspace
-    targetLatLng = map.unproject(targetPoint, targetZoom)
-
-  // Zoom/Pan center of map to offset location in worldspace
-  map.setView(targetLatLng, targetZoom)
+  if (points.length === 0) {
```

Comment:
just a note that this brought in/still has the `=== 0`

Reply:
Thanks, took care of it with some merge conflict cleanup!